system-addon/content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -161,7 +143,6 @@ export class TopSiteForm extends React.PureComponent {
 }
 
 TopSiteForm.defaultProps = {
-  label: "",
-  url: "",
+  TopSite: null,
   index: -1
```

Comment:
I don't think we need to keep around `defaultProps` anymore as we pass in the values from the one usage in `TopSites`. But it should also be safe to just keep it…